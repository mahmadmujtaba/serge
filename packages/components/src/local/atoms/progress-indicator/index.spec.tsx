/* global it expect */
import React from 'react'
import renderer from 'react-test-renderer'

import ProgressIndicator from './index'

const indicators = [
  { id: 1, complete: true },
  { id: 2, active: true },
  { id: 3 }
]
describe('ProgressIndicator component:', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<ProgressIndicator list={indicators} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
