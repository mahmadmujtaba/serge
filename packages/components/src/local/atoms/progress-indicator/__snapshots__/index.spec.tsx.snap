// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressIndicator component: renders correctly 1`] = `
<div
  className="progress-box"
>
  <div
    className="progress"
  >
    <div
      className="progress-indicator progress-indicator-complete"
    />
    <div
      className="progress-indicator progress-indicator-active"
    />
    <div
      className="progress-indicator"
    />
  </div>
</div>
`;
